### DISCUSSION TITLE
Add support for a way to search for text in the current view

### DISCUSSION THREAD

**Discussion by cipriancraciun** [zoidsh/meru#247] [Feature Requests] [answered]
As the title says, there isn't a way to search for a word (or phrase) in the current "view" (i.e. currently rendered HTML).  To be clear, I'm not speaking about the "search" functionality offered by GMail, but about the `Ctrl+F` that all browsers have.

---

It would be even better if there would be an extra option to only search the text of the currently displayed thread.

However a "find everywhere on the page" option is still priceless, especially for cluttered views like `Labels` or `Filters` for those that have many of those items.

**Comment by timche** [comment, marked as answer]
With v3 (#242) we have better possibilities to integrate this feature and I can see this being adding soon. So stay tuned!

**Comment by lorisilbagnino** [comment]
Is this feature in development? It would be very useful!

**Reply by timche** [reply]
Yes, it's planned! :) Will also share a product roadmap soon.
